modules/spring-config/src/main/java/org/mule/config/spring/parsers/specific/RetryPolicyDefinitionParser.java

```diff
@@ -23,6 +27,11 @@
  */
 public class RetryPolicyDefinitionParser extends OptionalChildDefinitionParser
 {
+
+    public static final String MULE_EXTENSION_NAMESPACE = "http://www.mulesoft.org/schema/mule/extension";
+    public static final String MULE_EXTENSION_PREFIX = "extension";
```

Comment:
all of these are already defined in SchemaConstant class. Try to reuse that. If any of these are missing there, then define them in that class instead